classes/filter_manager.php

```diff
@@ -57,19 +57,6 @@ public function __construct(block_xp_manager $manager) {
      *
      * @return array of fitlers.
      */
-    public function get_all_filters() {
```

Comment:
Because this method is `public`, I would prefer you do not remove it (also the PHP doc should have been removed). We could keep maintain the behaviour of this method as it was: "Returning all the filters to apply to the current course", and also keep `get_static_filters` even though the later wouldn't contain anything by default any more.